openreview/venue/matching.py

```diff
@@ -307,9 +309,7 @@ def _create_edge_invitation(self, edge_id, any_tail=False, default_label=None):
 
     def _build_conflicts(self, submissions, user_profiles, get_profile_info, compute_conflicts_n_years):
         if self.alternate_matching_group:
-            other_matching_group = self.client.get_group(self.alternate_matching_group)
-            other_matching_profiles = tools.get_profiles(self.client, other_matching_group.members, with_publications=True, with_relations=True)
-            return self._build_profile_conflicts(other_matching_profiles, user_profiles, compute_conflicts_n_years)
+            return
```

Comment:
Sorry if I'm missing where this is happening now - how are profile conflicts being built now?

Reply:
Profile conflicts are not being computed anymore. We don't allow SAC-AC conflicts to be computed from the request form. 
Do you need conflicts to be computed between two groups?

Reply:
Ohhh no I don't I didn't know we were deprecating that from the request form, okay that makes sense!

Reply:
I guess we could leave support for this just in case, but I removed it for now because I don't think the code is working. I would need to go over the code and fix it.